lib/dynflow/testing/in_thread_executor.rb

```diff
@@ -7,16 +7,16 @@ def initialize(world)
         @work_items = Queue.new
       end
 
-      def execute(execution_plan_id, finished)
+      def execute(execution_plan_id, finished, _wait_for_acceptance = true)
```

Comment:
Shouldn't there be a default value for `finished` to have the interface exactly the same as the `abstract` and `parallel` executors have?